raving-remoras/bangazon-workflow
# Rd edit employee

# Description
Given user is viewing an employee
When user clicks on the Edit link
Then user should be able to edit the last name of the employee
Or change the department to which the employee is assigned
Or change the computer assigned to the employee
Or add/remove training programs for the employee to attend in the future

## Number of Fixes
Added line to training.html per kelly's request

## Related Ticket(s)
#5 

## Steps to Test Solution

1. Go to an existing employee-- make sure you can edit and save all basic data (name, dept, if they're a supervisor, start and end date)
2. Find an employee with a computer-- assign them a different computer and make sure the new is created in the join table and that the old is marked as revoked
3. Add multiple trainings for an employee (ctrl+click for windows, cmd+click for mac), make sure they're now visible in the detail page
4. Remove multiple trainings for an employee-- make sure the correct trainings are deleted. Only future trainings should show in the edit form.

## Testing


## Documentation
